app/controllers/ServiceHourController.js

```diff
@@ -0,0 +1,38 @@
+
+(function() {
+  'use strict';
+
+  var models = require('../../models');
+  var ServiceHour = models.ServiceHour;
+  var Brother = models.Brother;
+
+  module.exports = {
+    findAll: function() {
+      return ServiceHour.findAll({
+        include: [Brother]
+      });
+    },
+    submit: function(brotherId, hours) {
+      hours.BrotherId = brotherId;
+      hours.state = 'u';
```

Comment:
Pull this out into two booleans: `looked_at` and `approved`. This way we aren't reliant on these characters